src/CarbonAware.CLI/src/CarbonAwareCLI.cs

```diff
@@ -77,9 +77,13 @@ private async Task<IEnumerable<EmissionsData>> GetEmissionsDataAsync(Dictionary<
 
```

Comment:
Quick question about PR/dev loop - do we prefer PR's to be focused solely on the issue, or is it ok to bundle up some potential optimizations/code tidy up as we go? For example, right above in GetEmissionsDataAsync, it looks like the same method is being called twice (and doesn't appear to do anything with the first returned set). Could tidy that up as I go, but is it preferred to cut a separate PR for optimizations?

Reply:
Hi Dylan, Thanks for jumping on this. 

On changes bundling, I think the general idea is to keep things small so they are easy to review. But I would say, if it is the same context and means the testing is not redundant thanks to that change, then I would probably include. 
From the sound of it here the change would be related and likely should be included (as it would involve the same testing and already remove some technical debt?).